Formula/django-completion.rb

```diff
@@ -1,14 +1,13 @@
 class DjangoCompletion < Formula
   desc "Bash completion for Django"
-  homepage "https://github.com/django/django"
+  homepage "https://www.djangoproject.com/"
   url "https://github.com/django/django/archive/3.1.5.tar.gz"
   sha256 "b481d9ffd70c51bd20039a3b5cd33ffe3ac188ba95d29b37e88e456472317eaf"
   license "BSD-3-Clause"
   head "https://github.com/django/django.git"
 
   livecheck do
-    url :head
-    regex(/^v?(\d+(?:\.\d+)+)$/i)
+    url :stable
```

Comment:
```suggestion
    url :stable
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

We still need the existing regex.